harness/wellKnownIntrinsicObjects.js

```diff
@@ -0,0 +1,458 @@
+  },
+  {
+    intrinsicName: "%SharedArrayBuffer%",
+    globalNameOrSource: "SharedArrayBuffer"
+  },
+  {
+    intrinsicName: "%SharedArrayBufferPrototype%",
+    globalNameOrSource: "SharedArrayBuffer.prototype"
+  },
+  {
+    intrinsicName: "%String%",
+    globalNameOrSource: "String"
+  },
+  {
+    intrinsicName: "%StringIteratorPrototype%",
+    globalNameOrSource: "Object.getPrototypeOf(new String()[Symbol.iterator]())"
+  },
+  {
+    intrinsicName: "%StringPrototype%",
+    globalNameOrSource: "String.prototype"
+  },
+  {
+    intrinsicName: "%Symbol%",
+    globalNameOrSource: "Symbol"
+  },
+  {
+    intrinsicName: "%SymbolPrototype%",
+    globalNameOrSource: "Symbol.prototype"
+  },
+  {
+    intrinsicName: "%SyntaxError%",
+    globalNameOrSource: "SyntaxError"
+  },
+  {
+    intrinsicName: "%SyntaxErrorPrototype%",
+    globalNameOrSource: "SyntaxError.prototype"
+  },
+  {
+    intrinsicName: "%ThrowTypeError%",
+    globalNameOrSource: "(function() { 'use strict'; return Object.getOwnPropertyDescriptor(arguments, 'callee').get })()"
+  },
+  {
+    intrinsicName: "%TypedArray%",
+    globalNameOrSource: "Object.getPrototypeOf(Uint8Array)"
+  },
+  {
+    intrinsicName: "%TypedArrayPrototype%",
+    globalNameOrSource: "Object.getPrototypeOf(Uint8Array).prototype"
+  },
+  {
+    intrinsicName: "%TypeError%",
+    globalNameOrSource: "TypeError"
+  },
+  {
+    intrinsicName: "%TypeErrorPrototype%",
+    globalNameOrSource: "TypeError.prototype"
+  },
+  {
+    intrinsicName: "%Uint8Array%",
+    globalNameOrSource: "Uint8Array"
+  },
+  {
+    intrinsicName: "%Uint8ArrayPrototype%",
+    globalNameOrSource: "Uint8Array.prototype"
+  },
+  {
+    intrinsicName: "%Uint8ClampedArray%",
+    globalNameOrSource: "Uint8ClampedArray"
+  },
+  {
+    intrinsicName: "%Uint8ClampedArrayPrototype%",
+    globalNameOrSource: "Uint8ClampedArray.prototype"
+  },
+  {
+    intrinsicName: "%Uint16Array%",
+    globalNameOrSource: "Uint16Array"
+  },
+  {
+    intrinsicName: "%Uint16ArrayPrototype%",
+    globalNameOrSource: "Uint16Array.prototype"
+  },
+  {
+    intrinsicName: "%Uint32Array%",
+    globalNameOrSource: "Uint32Array"
+  },
+  {
+    intrinsicName: "%Uint32ArrayPrototype%",
+    globalNameOrSource: "Uint32Array.prototype"
+  },
+  {
+    intrinsicName: "%URIError%",
+    globalNameOrSource: "URIError"
+  },
+  {
+    intrinsicName: "%URIErrorPrototype%",
+    globalNameOrSource: "URIError.prototype"
+  },
+  {
+    intrinsicName: "%WeakMap%",
+    globalNameOrSource: "WeakMap"
+  },
+  {
+    intrinsicName: "%WeakMapPrototype%",
+    globalNameOrSource: "WeakMap.prototype"
+  },
+  {
+    intrinsicName: "%WeakSet%",
+    globalNameOrSource: "WeakSet"
+  },
+  {
+    intrinsicName: "%WeakSetPrototype%",
+    globalNameOrSource: "WeakSet.prototype"
+  }
+];
+
+
+WellKnownIntrinsicObjects.forEach(wkio => {
+  var actual;
+
+  try {
+    actual = new Function("return " + wkio.globalNameOrSource)();
```

Comment:
perhaps, this should be documented somehow as a form of eval, mostly for uses of Test262 where eval parsing is not directly supported or limited.